SPECS/ocaml-sexplib/ocaml-sexplib.spec

```diff
@@ -0,0 +1,337 @@
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+%ifnarch %{ocaml_native_compiler}
+%global debug_package %{nil}
+%endif
+
+%bcond_with vim
```

Comment:
This seems unused and very confusing, since it seems like we're building without `vim` by default, whereas everywhere else we're clearly depending on `vim`. I'd just remove this line.

Reply:
Removed